apps/web/pages/connections.tsx

```diff
@@ -70,23 +69,6 @@ export default function ConnectionsPage(props: ServerSideProps) {
           )}
 
           <VeniceDatabaseSection />
-
-          {/* destinations column */}
-          {/* <section className="flex w-[24rem] shrink-0 flex-col gap-4">
```

Comment:
We can always get them back from git history. No need to keep inactive code via comment.